newdle/api.py

```diff
@@ -245,6 +247,21 @@ def get_newdles_participating():
     return NewdleParticipantSchema(many=True).jsonify(newdle)
 
 
+@api.route('/newdle/event', methods=('POST',))
+@use_kwargs(NewEventSchema(), locations=('json',))
+def create_event(title, duration, final_dt):
```

Comment:
What about POSTing to `/newdle/<code>/create-event` and then taking the data from the newdle's data?

Reply:
Do you mean to create the UUID (`<code>`) here then pass it on to /newdle/<code>/create-event (I'm not sure if that will be useful)? Can you clarify this?

Reply:
This API is not specific to any newdle atm - what I meant is to get rid of `NewEventSchema` altogether and use the `final_dt` and `duration` of the newdle you can get through its code.

Reply:
That's much cleaner, thanks! I'm now fetching all fields (`title`, `duration`, `final_dt`) from the Newdle object.